src/ripple/app/main/DBInit.cpp

```diff
@@ -92,30 +92,18 @@ const char* LedgerDBInit[] =
     );",
     "CREATE INDEX IF NOT EXISTS SeqLedger ON Ledgers(LedgerSeq);",
 
-    // InitialSeq field is the current ledger seq when the row
-    // is inserted. Only relevant during online delete
-    "CREATE TABLE IF NOT EXISTS Validations   (                   \
-        LedgerSeq   BIGINT UNSIGNED,                \
-        InitialSeq  BIGINT UNSIGNED,                \
-        LedgerHash  CHARACTER(64),                  \
-        NodePubKey  CHARACTER(56),                  \
-        SignTime    BIGINT UNSIGNED,                \
-        RawData     BLOB                            \
-    );",
-    "CREATE INDEX IF NOT EXISTS ValidationsByHash ON              \
-        Validations(LedgerHash);",
-    "CREATE INDEX IF NOT EXISTS ValidationsBySeq ON              \
-        Validations(LedgerSeq);",
-    "CREATE INDEX IF NOT EXISTS ValidationsByInitialSeq ON              \
-        Validations(InitialSeq, LedgerSeq);",
-    "CREATE INDEX IF NOT EXISTS ValidationsByTime ON              \
-        Validations(SignTime);",
+    // Old table and indexes no longer needed
+    "DROP TABLE IF EXISTS Validations;",
+    "DROP INDEX IF EXISTS ValidationsByHash;",
+    "DROP INDEX IF EXISTS ValidationsBySeq;",
+    "DROP INDEX IF EXISTS ValidationsByInitialSeq;",
+    "DROP INDEX IF EXISTS ValidationsByTime;",
```

Comment:
DROP INDEXes are redundant here. Table objects such as indices go away once the table goes away: https://www.sqlite.org/lang_droptable.html